example/package.json

```diff
@@ -23,9 +23,9 @@
   "dependencies": {
     "@react-navigation/native": "^6.0.16",
     "@react-navigation/native-stack": "^6.9.4",
-    "@tryvital/vital-core-react-native": "file:../packages/vital-core-react-native",
-    "@tryvital/vital-devices-react-native": "file:../packages/vital-devices-react-native",
-    "@tryvital/vital-health-react-native": "file:../packages/vital-health-react-native",
+    "@tryvital/vital-core-react-native": "link:../packages/vital-core-react-native",
+    "@tryvital/vital-devices-react-native": "link:../packages/vital-devices-react-native",
+    "@tryvital/vital-health-react-native": "link:../packages/vital-health-react-native",
```

Comment:
The example app now symlinks the packages, so changes in the packages are picked up by the example app immediately.

Reply:
Finally!